README.md

```diff
@@ -3532,6 +3532,12 @@ Manages a VRF for an OSPF router.
 | N9k-F    | 7.0(3)F1(1)        | 1.5.0                  |
 | N3k-F    | 7.0(3)F3(2)        | 1.8.0                  |
 
+#### <a name="cisco_ospf_vrf-caveats">Caveats</a>
+
+| Property | Caveat Description |
+|:-------------------|:-------------|
+| `redistribute`  | Minimum Module Version 1.11.0<br>No support for `redistribute maximum-prefixes` |

```

Comment:
Next release will be a major release so this should be `2.0.0`

Reply:
✔️